frontend/src/lib/components/LemonSelect.tsx

```diff
@@ -109,32 +112,32 @@ export function LemonSelect<O extends LemonSelectOptions>({
                     ref: popup?.ref,
                     overlay: sections.map((section, i) => (
                         <React.Fragment key={i}>
-                            {section.label ? (
-                                typeof section.label === 'string' ? (
-                                    <h5>{section.label}</h5>
+                            {section.title ? (
+                                typeof section.title === 'string' ? (
+                                    <h5>{section.title}</h5>
                                 ) : (
-                                    section.label
+                                    section.title
                                 )
                             ) : null}
-                            {Object.entries(section.options).map(([key, option]) => (
+                            {section.options.map((option, index) => (
                                 <LemonButton
-                                    key={key}
+                                    key={index}
                                     icon={option.icon}
                                     onClick={() => {
-                                        if (key != localValue) {
-                                            onChange?.(key)
-                                            setLocalValue(key)
+                                        if (option.value != localValue) {
+                                            onChange?.(option.value)
+                                            setLocalValue(option.value)
                                         }
                                     }}
                                     status="stealth"
                                     /* Intentionally == instead of === because JS treats object number keys as strings, */
                                     /* messing comparisons up a bit */
-                                    active={key == localValue}
+                                    active={option.value == localValue}
```

Comment:
I wonder... now that we aren't using object keys, can we remove the double equals?

Reply:
we can \o/